metals/src/main/scala/scala/meta/internal/metals/ServerCommands.scala

```diff
@@ -590,10 +590,16 @@ object ServerCommands {
     "Open the Metals logs to troubleshoot issues.",
   )
 
-  val OpenIssue = new OpenBrowserCommand(
-    "https://github.com/scalameta/metals/issues/new/choose",
-    "Open issue on GitHub",
-    "Open the Metals repository on GitHub to ask a question, report a bug or request a new feature.",
+  val OpenIssue = new Command(
+    "open-new-github-issue",
+    "Open an issue on GitHub",
+    "Open the Metals repository on GitHub to ask a question or report a bug.",
+  )
+
+  val OpenFeatureRequest = new OpenBrowserCommand(
```

Comment:
You might need to add it to the list of commands at the bottom of the file here, since they need to be registered at the start for the client to pick them up

Reply:
Now, those are only visible in the tree view. But we do want to make it possible to add it to the command pallet or smth. Is that what you mean?

Reply:
Ach, actually my bad. This is done via EchoCommand, but maybe it's good to let it work in Metals comamnds.